CODEOWNERS

```diff
@@ -17,6 +17,9 @@
 # C3 ownership
 /packages/create-cloudflare/ @cloudflare/c3
 
+# kv-asset-handler ownership
+/packages/kv-asset-handler/ @cloudflare/developer-advocacy @kristianfreeman @rickyrobinett @lauragift21 @LoganGrasby @craigsdennis
```

Comment:
Github seems to think some of these users/teams don't exist.

Reply:
Yes, they need to be added to this repo. cc. @lrapoport-cf